uint8 number1
uint8 number2
uint8 number3